obspy/io/mseed/tests/test_mseed_reading_and_writing.py

```diff
@@ -155,6 +155,7 @@ def test_downcast_int64_to_int32(self):
         # make sure the data can be written to mseed
         with io.BytesIO() as buf:
             st.write(buf, format="mseed")
+            buf.seek(0, 0)
```

Comment:
I don't understand how this worked before without the `seek()`, huh

Reply:
honestly, I thought the same when writing that line